docs/docs/introduction.md

```diff
@@ -22,7 +22,7 @@ To get the most out of Redwood, do two things:
 - [Join the community](https://redwoodjs.com/community)
 
 The tutorial is the best way to start your Redwood adventure.
-It's readable, feature-ful, and fun.
+It's readable, feature-full, and fun.
```

Comment:
I'm not 100% sure on this, but for now I'd prefer to it as-is because the real words that have "ful" suffixes only have one "l", like "beautiful, awful" etc
```suggestion
It's readable, feature-ful, and fun.
```